python/lsst/rapid/analysis/imageExaminer.py

```diff
@@ -60,7 +60,8 @@ class ImageExaminer():
                      "airmass": "Airmass",
                      "rotangle": "Rotation Angle",
                      "az": "Azimuth (deg)",
-                     "el": "Elevation (deg)"}
+                     "el": "Elevation (deg)",
```

Comment:
I wasn't sure how this change was being used (docstrings, please), but when I tracked it down, I wasn't sure why these are class variables instead of just variables in the `plot` method.  It's fine if you have a reason, but defining these here, but only using them once at the very end of the file is confusing to me.

Reply:
Yeah, tbh this is some of the more funky/whacky/hacky code. The whole class is designed to be initialised with an image and contain the results of its examination, the calculation of which is triggered on instantiation (which, if you think about object being the results of the exam, does make sense). But that's why it felt "OK" (just, kinda) to do things like shoving stuff into class state rather than passing it around. Make design much easier/more convenient, and given it's single use doesn't do any harm like it might in different objects.

Reply:
I am curious if you think that's an acceptable reason though.

Reply:
Considering this OK for now, so clicking resolve.